theme_aviato/views/images_library.xml

```diff
@@ -195,5 +195,12 @@
     <field name="name">website.library_image_02</field>
     <field name="url">/theme_aviato/static/src/img/content/s_wall_04.jpg</field>
 </record>
+<!-- Split Intro -->
+<record id="website.s_split_intro_default_image" model="ir.attachment">
```

Comment:
I think there is a small careless mistake here and in `theme_beauty`. You are overwriting the default record 👍